(DOCS-2753) Remove 'very' and 'highly' from docs (#12577)
DataDog/documentation: content/en/logs/log_collection/php.md

{{% /tab %}}
{{% tab "PHP Zend-Log" %}}

> Much of the useful information comes from additional context data that you can add to your logs and your events. Zend-Log makes this convenient by providing methods to set thread local context data that is then submitted automatically with all events. At any moment, log an event with contextual data:

Reviewer: ```suggestion
Useful logging information comes from additional context data that you can add to your logs and events. Zend-Log makes this convenient by providing methods to set thread local context data that is then submitted automatically with all events. At any moment, log an event with contextual data:
```
